Validator/Constraints/EmbeddedAddressFormatConstraint.php

```diff
@@ -24,7 +24,7 @@ public function __construct($options = null, $fieldOverrides = [])
   /**
    * {@inheritdoc}
    */
-  public function getTargets()
+  public function getTargets(): array|string
```

Comment:
I suggest not mixing types. If we return one type directly, there's no reason to hint at more than one.

Reply:
The parent from commerceguys/addressing sets this mixed type:  
https://github.com/commerceguys/addressing/blob/master/src/Validator/Constraints/AddressFormatConstraint.php#L40

If `EmbeddedAddressFormatConstraint.php` does not set the same return type, we have this Fatal Compile Error: Declaration of Daften\Bundle\AddressingBundle\Validator\Constraints\EmbeddedAddressFormatConstraint::getTargets() must be compatible with CommerceGuys\Addressing\Validator\Constraints\AddressFormatConstraint::getTargets(): array|string

symfony 7+ does the same:  
https://github.com/symfony/symfony/blob/7.0/src/Symfony/Component/Validator/Constraint.php#L283